src/styles.css

```diff
@@ -77,3 +76,58 @@ h1, h1, h3, h4, h5, h6 {
   color: var(--primary-color);
   background: var(--secondary-color);
 }
+
+.download-container {
```

Comment:
We can get rid of a lot of this css by using tailwind classes. Let me know if you need help with that.

Reply:
> Please add a description to this PR. Currently there's no description, which makes it hard to review:
> 
> <img alt="Screen Shot 2020-11-05 at 8 59 10 AM" width="1028" src="https://user-images.githubusercontent.com/6374832/98271796-2cd08680-1f45-11eb-8b07-d5158c130657.png">

@amilajack @hamirmahal Added description to PR, issue ticket info, and To-do's.

Reply:
> We can get rid of a lot of this css by using tailwind classes. Let me know if you need help with that.

For sure, will let you know if I need any help.
Thank you.